code/obj/item/gun/energy.dm

```diff
@@ -214,26 +216,185 @@ TYPEINFO(/obj/item/gun/energy)
 			projectiles += current_projectile
 
 
-////////////////////////////////////// Antique laser gun
-// Part of a mini-quest thing (see displaycase.dm). Typically, the gun's properties (cell, projectile)
-// won't be the default ones specified here, it's here to make it admin-spawnable (Convair880).
+//Antique laser gun
+
+//Maximum alpha a lens can be to fall into one of the damage tiers
+/// 180. glass, molitz, some other stuff
+#define CAPTAINGUN_T1_LENS_ALPHA_THRESHOLD 180
+/// 130. telecrystal, ectofiber, ice
+#define CAPTAINGUN_T2_LENS_ALPHA_THRESHOLD 130
+/// 80. starstone, rarely miraclium
+#define CAPTAINGUN_T3_LENS_ALPHA_THRESHOLD 80
+
+//Coil efficacy is scored as electrical conductivity + 2 if the material is an energy source
+//Minimum efficacy a coil can be to fall into one of the charge usage tiers
+/// 6. copper, pharosium, gold, etc.
+#define CAPTAINGUN_T1_COIL_EFFICACY_THRESHOLD 6
+/// 8. claretine, electrum, cerenkite, etc.
+#define CAPTAINGUN_T2_COIL_EFFICACY_THRESHOLD 8
+/// 10. custom alloy (figure it out, nerd)
+#define CAPTAINGUN_T3_COIL_EFFICACY_THRESHOLD 10
+
+//Damage dealt by different tiers of lens used
+/// 30. 2/3x normal laser damage
+#define CAPTAINGUN_T1_DAMAGE 30
+/// 45. normal laser damage
+#define CAPTAINGUN_T2_DAMAGE 45
+/// 60. 4/3x normal laser damage
+#define CAPTAINGUN_T3_DAMAGE 60
```

Comment:
These should just be different projectile types, with unique sprites as before.

Reply:
I could definitely look into having the projectile determine its sprite based on its damage, but having every possible combination of projectile stats be its own statically defined type would be very poor form. The projectiles have the exact same logic and just different variables, there's no reason to manually define each one when a constructor arg will do. 

And it prevents the existing situation where someone down the line takes the convenient route of reusing an existing projectile for their new gun, inadvertently linking the balance of two wholly unrelated things together. For example, the T1 and T3 captainguns currently use the strelka beams, meaning that the series of nerfs to the strelka pod gun has made them quite pathetic.